using-mobile-extensions/adobe-analytics/analytics-event-reference.md

```diff
@@ -353,21 +442,25 @@ Here is the definition of the key-value pairs in this event:
 
 **Forwarding the Analytics server response**
 
-```text
-{ "analyticsserverresponse": "example response to be forwarded" }
+```Javascript
+{
+  "analyticsserverresponse": "example response to be forwarded"
```

Comment:
Can you add two new fields here? `hitHost` and `hitUrl`?

`"hitHost": "the sever for the analytics hit"`

`"hitUrl": "the url for the analytics hit"`

I'd do it, but I don't want to cause merge conflicts with your PR :)

Reply:
no probs @nporter-adbe, done!